meta/sai_serialize.h

```diff
@@ -287,4 +290,8 @@ void sai_deserialize_queue_attr(
         _In_ const std::string& s,
         _Out_ sai_queue_attr_t& attr);
 
+void sai_deserialize_switch_attr(
```

Comment:
please remove we already have generic function to deserialize attributes for all objects